src/include/optimizer/statistics/top_k_elements.h

```diff
@@ -278,6 +241,82 @@ class TopKElements {
     return keys;
   }
 
+  /**
+   * Merge Top K Elements with another Top K Elements
+   * @param top_k_elements Top K Elements to merge with this
+   */
+  void Merge(const TopKElements<KeyType> &top_k_elements) {
+    sketch_->Merge(*(top_k_elements.sketch_));
+
+    for (auto &[key, count] : top_k_elements.entries_) {
```

Comment:
Hmm is it possible that the order in which the entries are added matters? Say the current min element will be incremented, but it is removed because a new element is added with a higher value first? Is this a concern even?

Reply:
I haven't traced through it to confirm this 100%, but I think the order could potentially lead to a different end state. However I think that's because it's a probabilistic data structure, and since it's a probabilistic data structure we can't expect it to be 100% correct. So I think that it's not a concern, what do you think?

Reply:
If I remember my suffering from taking Woodruff's class correctly, countminsketch is linear meaning (sketch each stream -> sum sketches) is the same as (concatenate streams -> sketch the concatenated stream).

Or in human, shouldn't be a concern.

Reply:
Both sound reasonable to me!